cmd/root.go

```diff
@@ -89,12 +90,15 @@ func database(cmd *cobra.Command, args []string) {
 	viper.Set("database.config", databaseConfig)
 }
 
-func logLevel(cmd *cobra.Command, args []string) error {
+func logLevel() error {
```

Comment:
was this method, and `database()` not using the arguments passed in previously?

Reply:
Yes. Neither of those were using the arguments so I removed them to clean up.